### DISCUSSION TITLE
add object

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46620] [Q&A] [answered]
Hello everyone ,
so i created this query :
![image](https://user-images.githubusercontent.com/101296742/172546591-f04521f4-80b5-4ead-8909-2dd46993d3cf.png)
And i want to use it like this :
![image](https://user-images.githubusercontent.com/101296742/172546764-3acb2ddb-c602-4110-99da-534110fefe6b.png)
its mean that when the query didn't return null we shouldn't add an object but when i call the route that should add it , it add it and i get this error :
**More than one result was found for query although one row or none was expected.**
Thank you !

**Comment by alessandro-podo** [comment]
that means you get more than one result. Then the function throws an exception.

**Comment by Elmatbaisaad** [comment, marked as answer]
yes , i resolve the probleme
![image](https://user-images.githubusercontent.com/101296742/173008857-b76682ff-95ad-40e8-95a6-d16d014e9428.png)
by doing this
